src/Docker/Manager/ContainerManager.php

```diff
@@ -615,4 +615,27 @@ public function kill(Container $container, $signal = "SIGKILL")
             throw UnexpectedStatusCodeException::fromResponse($response);
         }
     }
+
+    /**
+     * Rename a container (API v1.17)
+     *
+     * @param Container $container
+     * @param string $newname
+     *
+     * @throws \Docker\Exception\UnexpectedStatusCodeException
```

Comment:
I guess @hacfi's comment was more of a "note for next time"

The important thing in annotations is that an automated tool can get the FQCN either because it's already the FQCN (our case), or by inferring it using the `use` statements (which it could in this file since there's the correct use statement)